lib/middleware/check-progress.js

```diff
@@ -17,8 +18,7 @@ module.exports = function checkProgress(route, controller, steps, start) {
     var invalidatePath = function (req, res) {
         var potentialPaths = _.pluck(controller.options.forks, 'target')
             .concat(controller.options.next);
-
-        var nextStep = controller.getNextStep(req, res).replace(/\/edit$/, '');
+        var nextStep = FormController.prototype.getForkTarget.apply(controller, arguments);
```

Comment:
Do we really need to do this?

Can't we just do:
```js
var nextStep = controller.getForkTarget(req, res);
```

If I have a custom controller and I decide, for whatever crazy reason, to override the getForkTarget method (maybe I've invented a new style of forking, who knows) then I would expect my method to be used here, and presumably my method would _need_ to be used here.

And if it doesn't work? Well that's my lookout.